eng/pipelines/templates/stages/archetype-sdk-tool-azure-function.yml

```diff
@@ -0,0 +1,93 @@
+parameters:
+- name: ToolName
+  type: string
```

Comment:
Isn't string the default type? What value do we gain by explicitly listing the type? Not pushing back but just trying to understand when someone might want to do this.

Reply:
It is just me being explicit. I also prefix local member access in C# with ```this```. Don't hate me :) I'll switch it to being implcit.